backend/src/main/java/gov/cdc/usds/simplereport/service/TestOrderService.java

```diff
@@ -202,6 +219,49 @@ public int cancelAll() {
 	  return _repo.cancelAllPendingOrders(_os.getCurrentOrganization());
   }
 
+    @Transactional
+    @AuthorizationConfiguration.RequirePermissionUpdateTest
```

Comment:
This means that an entry-only user can mark a test as in error. Which isn't crazy except that they nominally can't see the completed tests at all...